09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    integrator.solvent_sorter = hoomd.mpcd.tune.ParticleSorter(trigger=20 * 20)
</pre><p>Use <code>trigger = integrator.collision_method.period * 20</code> to emphasize that sorting is performed every 20 collisions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>